### DISCUSSION TITLE
Multiple png files synthesize a single page CMYK color channel tif file

### DISCUSSION THREAD

**Discussion by 291598912** [ImageMagick/ImageMagick#6305] [Digital Image Processing]
Multiple PNG files were synthesized into a single page TIF file, and I wanted to convert it into a cmyk color channel. However, I found that there is a difference between the color and Photoshop color. What is the problem? The command is:
magick 1.png 2.png 3.png 4.png -density 300 -background transparent -colorspace cmyk -flatten -compress lzw  output.tif

**Comment by snibgo** [comment]
Perhaps Photoshop is using an ICC colour profile. With IM, see the `-profile` operation.

**Reply by 291598912** [reply]
May I ask how to use the ICC configuration file? When TIF provided it to me, there was no ICC file available. Should I use the ICC file from PS

**Reply by 291598912** [reply]
magick 1.png  2.png 3.png -profile sRGB.icc -density 300 -background transparent -colorspace cmyk -flatten -compress lzw  output.tif

This won't do either, there's still a color difference

**Comment by 291598912** [comment]
May I ask how to use the ICC configuration file? Give me the TIF file. If there is no ICC configuration file, do I need to use the ICC configuration file from PS

**Reply by 291598912** [reply]
CMYK configuration file, right? I'll go and take a look

**Comment by 291598912** [comment]
I found that there is no amount of cyan in cmyk. What's going on here

**Reply by 291598912** [reply]
The cmyk conversion has been resolved. Thank you
